packages/grpc-js/src/resolver-dns.ts

```diff
@@ -323,17 +331,23 @@ export function setup(): void {
   registerDefaultResolver(DnsResolver);
 }
 
+// camelCase needed for compatibility to grpc-native-js package
+// eslint-disable-next-line @typescript-eslint/class-name-casing
```

Comment:
This should actually be changed to `DnsUrl`. It's an internal type so that won't break anything. I just didn't notice when I was first writing the code and I haven't gotten around to fixing it.